conanfile.py

```diff
@@ -3,30 +3,39 @@
 
 from conans import ConanFile, tools
 import os
+import glob
 
 
 class sqlite_ormConan(ConanFile):
     name = "sqlite_orm"
-    version = "1.2"
+    version = "1.3"
     description = "SQLite ORM light header only library for modern C++."
     url = "https://github.com/bincrafters/conan-sqlite_orm"
     homepage = "https://github.com/fnc12/sqlite_orm"
+    topics = ("conan", "sqlite", "sql", "database", "orm", "sqlite_orm")
     author = "AlexandrePTJ <alpetitjean@gmail.com>"
     license = "BSD 3-Clause"
     exports = ["LICENSE.md"]
+    exports_sources = ["patches/*.diff"]
```

Comment:
where are these patches?

Reply:
I missed this one. No more patch are necessary, this line must be delete.